applications/liberland.md

```diff
@@ -0,0 +1,241 @@
+| 0c. | Testing Guide | We will have test files for all mayor functions, more about our testing procedure can be found under the testing section  |
+| 0d. | Docker | We will provide a Dockerfiles that can be used to run our node and also test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish a detailed article, walking a user through all functionality and features.)
+| 1. | Substrate chain | We will customize a substrate based code base and add our custom pallets |  
+| 2. | LLM module | We will create substrate pallet to handle the distribution and inflation of the LLM currency |  
+| 3. | Public Veto Module | We want our users to utilize our chain to have election and vote using a pallet  |  
+| 4. | Liberland Identity Pallet | We will create a substrate pallet that will allow accounts to have different account roles | 
+   
+   
+   
+#### Liberland Identity Pallet  
+Liberland's on-chain governance is tailor to serve a nation state, therefor we need functionality for users to be citizen, e-residence, assembly member, minister and so on. We will fork the identity pallet from Parity and rewrite it to fit our custom needs, in order to properly take part of the governance system.
+
+
+### Milestone 2: Land and Company registration
+
+- **Estimated duration:** 3 months
+- **FTE:**  3
+- **Costs:** 30 000 USD
+
+This milestone will allow citizens to register companies using our company register pallet, metaverse integration and a custom pallet that will handle land registration and management.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  MIT |
+| 0b. | Documentation | We will provide both inline code documentation as well as user guides that shows how people can interact with our pallets and chain |
+| 0c. | Testing Guide | We will have test files for all mayor functions, more about our testing procedure can be found under the testing section  |
+| 0d. | Docker | We will provide a Docker files that can be used to run our node and also test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish a detailed article, walking a user through all functionality and features.)
+| 1. | Land owner Pallet | We will create a pallet that will manage on-chain assets for land ownership |  
+| 2. | Metaverse integration Pallet | We will create a substrate pallet that interacts with our Metaverse|
+| 3. | Company Registration Pallet | We will create a substrate pallet that allows anyone that is holding LLM to incorporate a company | 
+
+
+We have currently had a very talented architect firm design and create our metaverse, that you can find more information about here:
+https://world.liberland.org/
+
+Our metaverse is currently being used for Virtual Reality Based conferences and social gatherings.
```

Comment:
Your metaverse is already live?! I didn't find any information about it on your [webpage](https://world.liberland.org/
).

Reply:
I added more documentation about this :)